specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2020-10-01/azurestackhci.json

```diff
@@ -0,0 +1,630 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ClusterList": {
+      "description": "List of clusters.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of clusters.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Cluster": {
+      "description": "Cluster details.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Cluster properties.",
+          "$ref": "#/definitions/ClusterProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ClusterProperties": {
+      "description": "Cluster properties.",
+      "type": "object",
+      "required": [
+        "aadClientId",
+        "aadTenantId"
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state.",
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Accepted",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "status": {
+          "description": "Status of the cluster agent.",
+          "type": "string",
+          "enum": [
+            "NotYetRegistered",
+            "ConnectedRecently",
+            "NotConnectedRecently",
+            "Disconnected",
+            "Error"
+          ],
+          "x-ms-enum": {
+            "name": "Status",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "cloudId": {
+          "description": "Unique, immutable resource id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "aadClientId": {
+          "description": "App id of cluster AAD identity.",
+          "type": "string"
+        },
+        "aadTenantId": {
+          "description": "Tenant id of cluster AAD identity.",
+          "type": "string"
+        },
+        "reportedProperties": {
+          "description": "Properties reported by cluster agent.",
+          "$ref": "#/definitions/ClusterReportedProperties"
+        },
+        "trialDaysRemaining": {
```

Comment:
does this property has constraint? If yes, could you pls add it in Swagger?

Reply:
I could add "minimum: 0"? I don't want to define upper bound.